examples/run_example.sh

```diff
@@ -5,19 +5,10 @@
 #   User
 #   Device
 
-function stage() {
-    BLUE_BOLD="\e[1m\e[34m"
-    RESET="\e[0m"
-    msg="$1"
-
-    echo
-    echo -e "$BLUE_BOLD$msg$RESET"
-}
-
 if [[ $1 != "" ]]; then
-    stage "Running single example: $1"
+    echo "Running single example: $1"
```

Comment:
Shouldn't this than use the "colored" print function?

Reply:
In previous review I got suggestion that coloring this certain one line is not required and to simplify this file as possible, and so I did.